demo/src/main/java/com/novoda/demo/MainActivity.java

```diff
@@ -132,7 +132,7 @@ public void onLicenseDownloaded(byte[] license) {
                 player = playerBuilder.build(getApplicationContext());
 
                 demoPresenter = new DemoPresenter(controllerView, player, player.getListeners(), playerView);
-                dialogCreator = new DialogCreator(getApplicationContext(), player);
+                dialogCreator = new DialogCreator(player);
```

Comment:
Why are we not passing in the context through the constructor anymore?

Reply:
ah yes, because it blows up. Current develop is broken and when you click on the video selector the application crashes. A Dialog needs an Activity.

Reply:
> A dialog needs an activity

Dialogs can be attached using a context, otherwise the builder would have complained. Probably it broke when we moved things around.

Reply:
well yeah but I mean the Activity context